packages/ai-gemini/src/tools/provider-tool-dispatch.test.ts

```
@@ -0,0 +1,66 @@
+import { describe, expect, it } from 'vitest'
+import {
+  codeExecutionTool,
+  computerUseTool,
+  fileSearchTool,
+  googleMapsTool,
+  googleSearchRetrievalTool,
+  googleSearchTool,
+  urlContextTool,
+} from './index'
+import { convertToolsToProviderFormat } from './tool-converter'
```

ESLint

[error] `./tool-converter` import should occur before import of `./index`

(import/order)